person.rb

```diff
@@ -1,16 +1,19 @@
-class Person
-  attr_reader :id
+require './nameable'
+
+class Person < Nameable
```

Comment:
Kindly implement a `correct_name` function in the Person call as per the requirements of this project. ⬇️
![image](https://user-images.githubusercontent.com/33268927/216042195-a24f712b-0fe8-474a-935c-3f4a95509c36.png)